srcpkgs/Converseen/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'Converseen'
+pkgname=Converseen
+version=0.9.8.1
+revision=1
+archs="i686* x86_64*"
+build_style=cmake
+configure_args="-DCMAKE_INSTALL_PREFIX=/usr"
+hostmakedepends="pkg-config"
+makedepends="qt5-devel qt5-tools-devel ImageMagick libmagick-devel"
+depends="qt5 ImageMagick ghostscript libheif libraw librsvg libwebp libwmf openexr hicolor-icon-theme"
```

Comment:
Required parts of qt5,  ImageMagick are detected, see generate-runtime-deps in build log. Other libs aren't used directly, but through ImageMagick, remove.